include/freerdp/settings.h

```diff
@@ -822,301 +846,301 @@ typedef struct _RDPDR_PARALLEL RDPDR_PARALLEL;
 struct rdp_settings
 {
 	/**
-	 * WARNING: this data structure is carefully padded for ABI stability!
-	 * Keeping this area clean is particularly challenging, so unless you are
-	 * a trusted developer you should NOT take the liberty of adding your own
-	 * options straight into the ABI stable zone. Instead, append them to the
-	 * very end of this data structure, in the zone marked as ABI unstable.
-	 */
+        * WARNING: this data structure is carefully padded for ABI stability!
+        * Keeping this area clean is particularly challenging, so unless you are
+        * a trusted developer you should NOT take the liberty of adding your own
+        * options straight into the ABI stable zone. Instead, append them to the
+        * very end of this data structure, in the zone marked as ABI unstable.
+        */
```

Comment:
@informatimago Can you change the script so that it retains the tabs for comments? Otherwise we'll run into troubles when using the code formatting script.

Reply:
Actually, the script leaves the comment as-is.  It was my editing that converted the tab to spaces in this comment. Sorry. I put back the tabs.